### ISSUE TITLE
Key to completely halt all (non-defensive) weapon fire

### ISSUE THREAD

**Issue by ReimeiSky** [endless-sky/endless-sky#645]
I want a key to completely halt all weapon fire, excluding defensive weapons (namely, anti-missile turrets).
Reason: I want to disable Korath ships without my fleet annihilating their worthless fighters (as well as the ships themselves).

**Comment by thenerdfreak** [comment]
Quick tip: Take out their fighters first. Wait till they come out. If you have anything better than or equal to a heavy laser it's like a split second and they're toast and you won't fry your target

**Comment by Bladewood** [comment]
I'm still hoping for this to happen...

**Comment by KiLEdEnNis** [comment]
a hold fire (similar to hold position) key would be a great addition to be honest

and imma put this here because this gave me the idea :3
(an enhancement, similar to this, where in a battle between 2+ factions, a faction "surrenders" after a certain amount of losses, where they turn friendly, that would also be cool and the hold fire button would be great for that)

**Comment by Bladewood** [comment]
Putting that _here_, in particular, is a horrible idea, because it is so far back in the archive that nobody will ever see it.

**Comment by KiLEdEnNis** [comment]
if you really want it to happen, make a new issue then

**Comment by KevinKorzekwa** [comment]
Only a tiny bit related, and I'll throw it here because you two are active, but I have always wanted to be able to toggle on and off firing for my fleet.

**Comment by KiLEdEnNis** [comment]
a.k.a. hold fire mode
MZ please D:

**Comment by TELunus** [comment]
Sorry to necro, but what ever happened to this?  I see Mike closed it, but I don't get why.  Was there a decision not to do this?  If so what was the reason for that decision?  Or did it get added to the game and I'm just not seeing it?

**Comment by ThrawnCA** [comment]
> Quick tip: Take out their fighters first. Wait till they come out. If you have anything better than or equal to a heavy laser it's like a split second and they're toast and you won't fry your target

That may work up to a point, but you can still be faced with a situation where you're just about to loot a disabled Palavret for its jump drive, and then a new world ship jumps in and your trigger-happy escorts shoot *through* the Palavret to get to it. It's happened to me lots of times.

**Comment by ThrawnCA** [comment]
This is particularly relevant in missions like  "Wanderers: Sestor: Exiles 2" where you encounter hostile ships but you aren't supposed to shoot back. The only way I was able to complete that one was to park all my escorts, because otherwise they would promptly blow the attackers to bits and fail the mission.

**Comment by Zitchas** [comment]
Yeah,. Having a key we could press to completely hold fire would be very appreciated.
